cmd/lifecycle/exporter.go

```diff
@@ -33,6 +34,9 @@ type exportCmd struct {
 
 	//flags: paths to write outputs
 	analyzedPath string
+
+	//construct before dropping privileges
+	keychain authn.Keychain
```

Comment:
Same comment as https://github.com/buildpacks/lifecycle/pull/409/files#r540433488
But here we need to pass the keychain as an argument to two functions: `export` and `initRemoteImage`.